.github/workflows/build-test-cxx.yml

```diff
@@ -78,9 +78,9 @@ jobs:
       uses: lukka/get-cmake@v3.29.0
 
     - name: 'Specific XCode version'
-      if: matrix.os == 'macos-12'
+      if: matrix.os == 'macos-13'
       run: |
-        sudo xcode-select -s "/Applications/Xcode_13.2.1.app"
+        sudo xcode-select -s "/Applications/Xcode_14.3.1.app"
```

Comment:
We need to keep the same XCode version.

Reply:
Keeping the old version does not work, though: `xcode-select: error: invalid developer directory '/Applications/Xcode_13.2.1.app'`, as shown here:
https://github.com/InsightSoftwareConsortium/ITKIOOMEZarrNGFF/actions/runs/11392959753/job/31700210355?pr=75

Reply:
I am not seeing XCode 13 available in the macOS 13 images:

https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#xcode

I guess we have to make the change, but it may cause issues.

Reply:
It may then be sensible to ensure the relevant deployment target value is propagated ?

Reply:
Yes, we need to keep the same osx deployment target, but that should not change.